pootle/apps/pootle_misc/checks.py

```diff
@@ -1004,18 +1004,19 @@ def get_qualitycheck_schema(path_obj=None):
     checks = get_qualitychecks()
 
     category_names = {
-        Category.CRITICAL: _("Critical"),
-        Category.COSMETIC: _("Cosmetic"),
-        Category.FUNCTIONAL: _("Functional"),
-        Category.EXTRACTION: _("Extraction"),
-        Category.NO_CATEGORY: _("Other"),
+        Category.CRITICAL: "Critical",
+        Category.COSMETIC: "Cosmetic",
+        Category.FUNCTIONAL: "Functional",
+        Category.EXTRACTION: "Extraction",
+        Category.NO_CATEGORY: "Other",
     }
 
     for check, cat in checks.items():
         if cat not in d:
             d[cat] = {
                 'code': cat,
-                'title': u"%s" % category_names[cat],
+                'name': category_names[cat].lower(),
+                'title': u"%s" % _(category_names[cat]),
```

Comment:
This doesn't work, the localizable names above should be kept. Feel free to add an extra mapping for retrieving identifiers.